.drone.yml

```diff
@@ -1,7 +1,107 @@
 ---
 kind: pipeline
 type: kubernetes
-name: docs-pr
+name: pr build
+
+trigger:
+  event:
+  - pull_request
+
+steps:
+  # https://discourse.drone.io/t/how-to-exit-a-pipeline-early-without-failing/3951
+  # If a change is entirely documentation, skip the expensive build & test steps.
+  - name: short circuit docs changes
+    image: docker:git
+    commands:
+    -  ./build.assets/drone/diff-is-all-docs.sh $DRONE_COMMIT_BEFORE..$DRONE_COMMIT_AFTER && exit 78 || exit 0
```

Comment:
Is there any significance in using exit code 78?

Reply:
Yeah, https://discourse.drone.io/t/how-to-exit-a-pipeline-early-without-failing/3951 has more info.  I (or rather Gus) tried to cover that in the comment above, but perhaps it is too far from the exit to be contextually relevant.

Reply:
Ah, thanks for the clarification.